filesig.cabal

```diff
@@ -51,10 +51,10 @@ category:            System
 -- Extra files to be distributed with the package, such as examples or a
 -- README.
 extra-source-files:
-    CHANGELOG.md,
-    README.md,
-    test/fixtures/doc.doc,
-    test/fixtures/rtf.rtf,
+    CHANGELOG.md
```

Comment:
Oh it's great to know I don't need commas if they're on new lines!

Reply:
You don't need commas here, no. I'm not sure you ever need commas, but I think I do see them used in the `build-depends:` most of the time. But it's really easy to test, because you get errors if they're required :P